bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -303,8 +303,8 @@ protected function getTreeNodeConfig($element): array
 
         if ($tmpObject['leaf']) {
             $tmpObject['expandable'] = false;
+            $tmpObject['leaf'] = false; //this is required to allow drag&drop
```

Comment:
we have to use this due to ExtJS limitation, which doesn't allow drag&drop on leaf nodes.